packages/design/src/scripts/tokenServices/tokenConstants.ts

```diff
@@ -0,0 +1,2 @@
+export const baseUnit = "16px";
+export const MAX_DEPTH = 200;
```

Comment:
Done! At some point I was manually keeping track of recursion depth to prevent the error, but then I figured it just made more sense to let the error happen and catch it instead.